Assets/Scripts/Gamestate/Round.cs

```diff
@@ -7,7 +7,7 @@
 public class Round
 {
     private PlayerManager winner;
-    public PlayerManager Winner => winner;
+    public PlayerIdentity Winner => winner.identity;
```

Comment:
Why continue to have a PlayerManager in the private field? We only know the winner after the round is over, and by that point PlayerManager should be mostly irrelevant.

Reply:
But that doesn't matter for the winner, right? We need PlayerManagers for other things, but for the winner, the only thing that matters is the identity?

(anyways, this is some nitpickery since the winner that is publicly available *is* an identity)

Reply:
Yup, updated for clarity